python/pynessie/commands/_branch_tag_handlers.py

```diff
@@ -99,16 +101,33 @@ def _handle_json_output(input_data: list, ref_name: str) -> str:
     return ReferenceSchema().dumps(input_data, many=True)
 
 
-def _handle_normal_output(input_data: list, verbose: bool, default_branch: str) -> str:
+def _handle_normal_output(input_data: list, verbose: bool, default_branch: str, show_additional_info: bool = False) -> str:
     output = ""
     max_width = max((len(i.name) for i in input_data), default=0)
     for x in input_data:
+        additional_info = ""
+        if show_additional_info and x.metadata:
+            num_commits_ahead = x.metadata.num_commits_ahead if x.metadata.num_commits_ahead else "?"
+            num_commits_behind = x.metadata.num_commits_behind if x.metadata.num_commits_behind else "?"
+            num_total_commits = x.metadata.num_total_commits if x.metadata.num_total_commits else "?"
+            common_ancestor_hash = x.metadata.common_ancestor_hash if x.metadata.common_ancestor_hash else "?"
+            head = (
+                "[author: {} | commitTime: {} | msg: {}]".format(
```

Comment:
Is `msg` that useful? I imagine it can easily spam STDOUT... maybe abbreviate it to N chars or the first new line?

Reply:
I wasn't sure about which parts of a commit would be useful here either, but I figured if you're asking for the additional metadata, then the msg might be useful, although I don't have any strong opinions here, so any suggestion is welcome

Reply:
I added some logic to abbreviate the msg if it gets longer than 35 chars